### PR TITLE
[Feature] Add Zoom to Fit Waypoints Shortcut

### LINKED ISSUE #349: Shortcut Control + f with zoom to fit trajectory
Shortcut `control + f` for "fitting" a trajectory by zooming to it.

### PR DESCRIPTION
**Pull request by Awesomeplayer165**
Closes #349

### CODE REVIEW

**Comment by Awesomeplayer165** [comment]
Was wondering if a toast should actually be shown if there are no waypoints to zoom to?

File: src/document/DocumentManager.ts [hunk 1]
```diff
@@ -205,6 +205,13 @@ export class DocumentManager {
     hotkeys("command+n,ctrl+n", { keydown: true }, () => {
       this.newFile();
     });
+    hotkeys("command+f,ctrl+f", () => {
```
Comment (calcmogul) [inline src/document/DocumentManager.ts:208]: Isn't `Ctrl + F` typically search?

Comment (calcmogul) [reply src/document/DocumentManager.ts:208]: Ctrl + 0 is common for default zoom. Is "zoom to fit waypoints" effectively our default?

Comment (Awesomeplayer165) [reply src/document/DocumentManager.ts:208]: Yes!

**Review by calcmogul — APPROVED** [review]

**Pull request merged**